src/views/Dashboard/components/Widgets/SendWidget/SendWidget.tsx

```diff
@@ -31,22 +30,34 @@ const SendWidget: React.FC = () => {
     setSendQuantity(quantityInput);
   }, []);
 
+  const handleFetch = useCallback(() => {
+    console.log("fetch not implemented in sendWidget");
+  }, []);
+
   return (
     <>
       <StyledWidgetHeading>Send JUP</StyledWidgetHeading>
 
       <Grid container>
         <Grid item xs={10}>
-          <StyledAutocomplete
-            freeSolo
-            options={placeHolderVals.map((option) => option)}
-            renderInput={(params) => <TextField {...params} label="Enter asset name" />}
-          />
-          <StyledToAddressInput onChange={(e) => handleToAddressEntry(e.target.value)} placeholder="To Address" />
-          <StyledQuantityInput onChange={(e) => handleQuantityEntry(e.target.value)} placeholder="Quantity" />
+          <Stack sx={{ width: "95%", margin: "10px", padding: "10px" }}>
+            <JUPAssetSearchBox fetchFn={handleFetch} />
```

Comment:
there's an asset search box here but the widget only supports sending JUP. either change it to a send jup only widget or extend it to support assets (probably as a stretch goal for now)

Reply:
removed for now, re-styled the send button to better utilize the widget height